client/state/ui/editor/selectors.js

```diff
@@ -33,10 +33,11 @@ export function isEditorNewPost( state ) {
 /**
  * Returns the editor URL for duplicating a given site ID, post ID pair.
  *
- * @param  {Object}  state Global state tree
+ * @param  {Object}  state      Global state tree
```

Comment:
the word state looks like it has a single extra space before it if you want it lined up with the rest of the var names